applications/luci-app-csshnpd/htdocs/luci-static/resources/view/sshnpd/enroll.js

```diff
@@ -0,0 +1,97 @@
+'use strict';
+'require view';
+'require dom';
+'require fs';
+'require ui';
+'require uci';
+'require network';
+
+
+return view.extend({
+	handleCommand: function(exec, args) {
+		var buttons = document.querySelectorAll('.diag-action > .cbi-button');
+
+		for (var i = 0; i < buttons.length; i++)
+			buttons[i].setAttribute('disabled', 'true');
+
+		return fs.exec(exec, args).then(function(res) {
+			var out = document.querySelector('textarea');
+
+			dom.content(out, [ res.stdout || '', res.stderr || '' ]);
+		}).catch(function(err) {
+			ui.addNotification(null, E('p', [ err ]))
+		}).finally(function() {
+			for (var i = 0; i < buttons.length; i++)
+				buttons[i].removeAttribute('disabled');
+		});
+	},
+
+	handleEnroll: function() {
+		return this.handleCommand('at_enroll.sh', "");
+	},
+
+	load: function() {
+		return uci.load('sshnpd').then(function() {
+			var atsign = uci.get_first('sshnpd','','atsign'),
+				keyfile = '/root/.atsign/keys/'+atsign+'_key.atKeys';
+				return L.resolveDefault(fs.stat(keyfile), {});
+		});
+	},
+
+	render: function(res) {
+
+		var has_atkey = res.path,
```

Comment:
This does not correspond to the standard of look and feel and is also difficult to read.
Can you please define more then one variable  with `var` .
Please replace `var` with `let`.
See we have:
```
let has_atkey = res.path;
let atsign = uci.get_first('sshnpd','','atsign');
let device = uci.get_first('sshnpd','','device');
			
```

Cod is always read more often than written and I think it is difficult to make a change because everything is done in one line at the end.